consensus/common/src/test/java/org/hyperledger/besu/consensus/common/MigratingMiningCoordinatorTest.java

```diff
@@ -122,28 +123,16 @@ public void stopShouldUnregisterThisCoordinatorAsObserver() {
   }
 
   @Test
-  public void onBlockAddedShouldMigrateToNextDelegateAndRemoveItAsObserver() {
-    final BftMiningCoordinator delegateCoordinator = createDelegateCoordinator();
-    when(blockHeader.getNumber()).thenReturn(MIGRATION_BLOCK_NUMBER - 1);
-    when(blockchain.observeBlockAdded(delegateCoordinator)).thenReturn(2L);
-
-    new MigratingMiningCoordinator(
-            createCoordinatorSchedule(coordinator1, delegateCoordinator), blockchain)
-        .onBlockAdded(blockEvent);
-
-    verify(blockchain).observeBlockAdded(delegateCoordinator);
-    verify(blockchain).removeObserver(2L);
-  }
-
-  @Test
-  public void onBlockAddedShouldMigrateToNextMiningCoordinatorAndDelegate() {
+  public void onBlockAddedShouldNotThrowExceptionWhenMigrating() {
```

Comment:
Don't think this test is worth having as it is

Reply:
Removed